python/ray/util/client/server/dataservicer.py

```diff
@@ -17,10 +17,12 @@
 
 
 class DataServicer(ray_client_pb2_grpc.RayletDataStreamerServicer):
-    def __init__(self, basic_service: "RayletServicer"):
+    def __init__(self, basic_service: "RayletServicer",
```

Comment:
I think an easy way to test this would be following test_client_init.py::test_num_clients. You can connect and disconnect clients, and check the job id is refreshed after you hit zero clients.

Reply:
+1 test_client_init is perfect for this purpose